browser/templates/browser/investigator.html

```diff
@@ -79,6 +79,26 @@
         </div>
     </div>
     {% endwith %}
+
+    {% with form.institution as field %}
+    <div class="row">
+        <div class="col-xs-12">
+            {% with form.institution as field %}
```

Comment:
why do you declare this again?

Reply:
I had put in an extra form field declaration which is not needed. So, have removed it.